packages/instantsearch-react/examples/history.js

```diff
@@ -2,7 +2,17 @@ import {useRouterHistory} from 'react-router';
 import {createHistory} from 'history';
 import qs from 'qs';
 
+function alphabeticalSort(a, b) {
```

Comment:
Oh wow. Are there cases where we might need the specific behavior of localeCompare? (sorting strings based on locale)